injected/src/features/message-bridge.js

```diff
@@ -0,0 +1,168 @@
+import { Messaging } from '@duckduckgo/messaging';
+import ContentFeature from '../content-feature';
+import {
+    InstallProxy,
+    ProxyNotification,
+    ProxyRequest,
+    ProxyResponse,
+    SubscriptionRequest,
+    SubscriptionResponse,
+    SubscriptionUnsubscribe,
+} from './message-bridge/schema.js';
+
+/**
+ * This part has access to messaging handlers
+ */
+export class MessageBridge extends ContentFeature {
```

Comment:
Everything in this class is running within the isolated world on macOS. If we know that windows/android will not, then we'll need to do some globals-capturing here too.